understanding/22/focus-not-obscured-minimum.html

```diff
@@ -20,33 +21,34 @@ <h2>Focus Not Obscured Success Criterion text</h2>
         <blockquote>
             <p>When a <a>user interface component</a> receives keyboard focus, the component is not entirely hidden due to author-created content.</p>
         </blockquote>
+        <p class="note">If the interface is configurable so that the user can reposition content such as toolbars and non-modal dialogs, then only the initial positions of user-movable content would be considered for testing and conformance of this Success Criterion.</p>
      </section>
    
     <section id="intent">
         
         <h2>Intent of Focus Not Obscured</h2>
```

Comment:
Not introduced by this PR, but noticing during review, this probably should have `(Minimum)` at the end to match the SC title

Reply:
The SC name is not in the note. "Success Criterion" is a generic phrase which is capitalized by convention in WCAG materials.